packages/analytics-js/src/services/ErrorHandler/processError.ts

```diff
@@ -14,6 +16,21 @@ const processError = (error: SDKError): string => {
       errorMessage = error;
     } else if (error instanceof Error) {
       errorMessage = error.message;
+    } else if (error instanceof Event) {
+      const eventTarget = error.target as EventTarge;
```

Comment:
is EventTarge a typo missing last letter?

Reply:
Yes

Reply:
Addressed